.werft/platform-delete-preview-environment.yaml

```diff
@@ -24,7 +24,7 @@ pod:
         secretName: harvester-vm-ssh-keys
   containers:
     - name: build
-      image: eu.gcr.io/gitpod-core-dev/dev/dev-environment:pd-dev-image-gcloud.2
+      image: eu.gcr.io/gitpod-core-dev/dev/dev-environment:aledbf-upfi.0
```

Comment:
Could you confirm this update is ok?

This was the only one that was not a "_mads-update-leeway.0 -> aledbf-upfi.0_" change. So I'm not sure if it was missed on last update or if it's a mistake on this update.

Reply:
Should be fine, my bad for overlooking it when I did my upgrade ☺️